drivers/hmis/spec/requests/hmis/project_config_spec.rb

```diff
@@ -47,28 +47,51 @@
     GRAPHQL
   end
 
+  let(:get_project_configs) do
+    <<~GRAPHQL
+      query GetProjectConfigs {
+        projectConfigs {
+          nodesCount
+          nodes {
+            id
+          }
+        }
+      }
+    GRAPHQL
+  end
+
   let!(:c1) { create :hmis_hud_client, data_source: ds1 }
-  describe 'when the user has access' do
-    let!(:access_control) { create_access_control(hmis_user, p1) }
+  let!(:access_control) { create_access_control(hmis_user, p1) }
 
-    it 'should successfully create a new auto enter config' do
-      mutation_input = { config_type: 'AUTO_ENTER', project_type: 'ES_ENTRY_EXIT' }
-      response, result = post_graphql(input: mutation_input) { create_project_config }
-      expect(response.status).to eq(200), result.inspect
-      project_config_id = result.dig('data', 'createProjectConfig', 'projectConfig', 'id')
-      expect(project_config_id).not_to be_nil
-      project_config = Hmis::ProjectConfig.find(project_config_id)
-      expect(project_config.class.name).to eq('Hmis::ProjectAutoEnterConfig')
-      expect(project_config.project_type).to eq(0)
-    end
+  it 'should successfully create a new auto enter config' do
```

Comment:
These first 2 tests are substantively unchanged. I just restructured them by making use of `remove_permissions`. This way we can share `access_control` at the top level across all 3 tests. The 3rd test beginning on line 83 is the new test case added in this PR.